README.md

```diff
@@ -55,15 +56,18 @@ Execute the command with the required inputs.
 ### Docker image
 
 ```shell
-docker run -it --rm -v$(PWD):/code \
+docker run -it --rm \
+  -v $(PWD):/code \
+  -v $(PWD)/output/:/output \
   -e DOCPATH='/code' \
+  -e OUTPATH='/output' \
   -e NAME='' \
   -e MERGETYPE='' \
   -e AUTHOR='' \
   -e EMAIL='' \
   -e DOCNAMESPACE='' \
   -e FILETYPE='' \
-  docker.io/philipssoftware/spdxmerge:v0.1.0
+  docker.io/philipssoftware/spdxmerge:v0.1.1
```

Comment:
This is a Chicken-egg problem.. We will create a new release and will update this accordingly.